### ISSUE TITLE
Covenant Antimatter Charge should have a tracker

### ISSUE THREAD

**Issue by SharkLordSatan** [HaloSpaceStation/HaloSpaceStation13#2826]
One major thing I've noticed occuring during Base Assault is that as soon as the marines get their hands on the Antimatter Charge, they'll hide it somewhere so that the covenant players have no clue where it is.

I feel like this is unfair since the Covenant have no means of knowing where the bomb is (unless they metagame and tell their team where it is by finding it while ghosted), so I propose a solution, to encourage the Covenant to keep up their assault and try and GIT DAT FUKKIN BOMB even after a defeat, so that its' not a "soft" round loss as soon as the Antimatter Charge is hidden.

Basically; a Pinpointer that points you in the direction of the bomb once you're on the same map/z-level as it. Functions exactly the same as literally any other Pinpointer, but it's solely for locating the Antimatter Charge.

If we had such a Pinpointer/tracker available I feel like it'd hopefully encourage some Covenant players to try and get their bomb back so that they can resume the mission.

**Comment by SharkLordSatan** [comment]
Some other thoughts on this:

- I'd give a decent but low-ish amount per species equipment vendor, probably about 10. Not sure if they should be free or 1 req point.
- Humans/UNSC should **NOT** get a tracker at all, as it'd basically allow them to not only tell where the bomb is, but also tell where an incoming Covenant attack is if they decided to bring in the bomb with them. (Even then, they can just grab a Pinpointer off some grunt's corpse or something).

Basically; once the humans get the bomb, they can't just hide it and then wait for the round to end. It'd be up to them to ensure that the bomb is actually defended and does NOT fall back into the hands of the Covenant, as well as to give the Covenant a chance to try and retrieve it again (if they so choose) since they'll be able to actually find the bomb.

**Comment by Aroliacue** [comment]
Antimatter bomb can be respawned, tracker not really relevant anymore. Can be closed
